max.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"unicode/utf8"
+)
+
+func max(x []string) string {
+	var sliceLen, p int
+	if len(x) <= 0 {
+		fmt.Print("Slice has no elements")
+		os.Exit(1)
+	}
+	sliceLen = len(x)
+	maxCharLen := utf8.RuneCountInString(x[0])
+	for i := 1; i < sliceLen; i++ {
+
+		if maxCharLen < utf8.RuneCountInString(x[i]) {
```

Comment:
Readability can be improved save result of utf8.RuneCountInString(x[I]) to variable and compare two variables